plugin/evm/ftso_system_test.go

```diff
@@ -0,0 +1,221 @@
+package evm
+
+import (
+	"math"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/flare-foundation/coreth/accounts/abi"
+	"github.com/flare-foundation/coreth/accounts/abi/bind"
+	"github.com/flare-foundation/coreth/accounts/abi/bind/backends"
+	"github.com/flare-foundation/coreth/core"
+)
+
+var (
+	// VotePower contract
+	testAbiVotePowerJSON = `[{"inputs":[{"internalType":"uint256","name":"_vpInt","type":"uint256"}],"stateMutability":"nonpayable","type":"constructor"},{"inputs":[{"internalType":"address","name":"provider","type":"address"}],"name":"votePowerOf","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"},{"inputs":[],"name":"vpInt","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"view","type":"function"}]`
+	testAbiVotePowerBin  = `0x608060405234801561001057600080fd5b506040516101493803806101498339818101604052602081101561003357600080fd5b8101908080519060200190929190505050806000819055505060ef8061005a6000396000f3fe6080604052348015600f57600080fd5b506004361060325760003560e01c8063142d10181460375780638966b43214608c575b600080fd5b607660048036036020811015604b57600080fd5b81019080803573ffffffffffffffffffffffffffffffffffffffff16906020019092919050505060a8565b6040518082815260200191505060405180910390f35b609260b3565b6040518082815260200191505060405180910390f35b600080549050919050565b6000548156fea264697066735822122043d0eedff37b596f65fc5d3f9b98f8444bac96fcf123c979b81ece56c2991d2664736f6c63430007060033`
+	testAbiVotePower, _  = abi.JSON(strings.NewReader(testAbiVotePowerJSON))
```

Comment:
ABIs should be decoded as part of an `init` function or a test setup function, so that we can actually check they happened correctly. Otherwise, we might have silent errors.